cli.js

```diff
@@ -44,17 +42,37 @@ wss.on('connection', (ws) => {
   shareDBBackend.listen(new WebSocketJSONStream(ws));
 });
 
+// app.get('/', (req, res) => {
+//   res.send('Hello World!');
+// });
+
 // Serve static files
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
 const dir = path.join(__dirname, '/dist');
 
 app.use(express.static(dir));
 
-// Create initial document
 const shareDBDoc = shareDBConnection.get('documents', '1');
 shareDBDoc.create(initialDocument, json1.type.uri);
 
+shareDBDoc.subscribe(() => {
```

Comment:
Nice as a first pass!

I think it can be simplified such that we don't need to look at the ops at all.

Also, eventually we should debounce. Currently this will save the file on each keystroke, which is too frequent.

Here's a sketch of an idea:

 * On each op, set a `dirty` flag and start a timer
 * On each op while the timer is going, reset the timer
 * When the timer finishes, write only the files that changed
 * We can detect which files have changed by storing in memory the state of the files as they were the last time a save happened and comparing each one